schemas/db-schema.sql

```diff
@@ -0,0 +1,22 @@
+CREATE DATABASE todo
+    CHARACTER SET = 'utf8'
+    COLLATE = 'utf8_polish_ci';
+
+
+use todo;
+
+
+CREATE TABLE users
+(
+user_id INT(6) NOT NULL PRIMARY KEY,
+username VARCHAR(30)
+);
+
+CREATE TABLE tasks
+(
+task_id INT(6) NOT NULL PRIMARY KEY AUTO_INCREMENT,
+user_id INT(6) NOT NULL,
```

Comment:
You should specify the [foreign key constraint](https://mariadb.com/kb/en/constraint/) for this column